testsuite/testsuite_impl/my_test/my_test_.go

```diff
@@ -0,0 +1,202 @@
+package my_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/kurtosis-tech/kurtosis-client/golang/lib/networks"
+	"github.com/kurtosis-tech/kurtosis-client/golang/lib/services"
+	"github.com/kurtosis-tech/kurtosis-testsuite-api-lib/golang/lib/testsuite"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+const (
+	ethereumGoClientImageName = "ethereum/client-go"
+	ethereumGoClientPort      = 8565
+	rpcPort                   = 8545
+	adminInfoRpcCall          = `{"jsonrpc":"2.0","method": "admin_nodeInfo","params":[],"id":67}`
+
+	waitForStartupTimeBetweenPolls = 1 * time.Second
+	waitForStartupMaxPolls         = 90
+
+	serviceID = "my-eth-client"
+)
+
+type MyTest struct{}
+
+type NodeInfoResponse struct {
+	Result NodeInfo `json:"result"`
+}
+
+type NodeInfo struct {
+	Enode string `json:"enode"`
+}
+
+func (test MyTest) Configure(builder *testsuite.TestConfigurationBuilder) {
+	builder.WithSetupTimeoutSeconds(360).WithRunTimeoutSeconds(360)
+}
+
+func (test MyTest) Setup(networkCtx *networks.NetworkContext) (networks.Network, error) {
+	containerCreationConfig, runConfigFunc := getMyServiceConfigurations()
+
+	serviceContext, hostPortBindings, err := networkCtx.AddService(serviceID, containerCreationConfig, runConfigFunc)
```

Comment:
It looks like we only add one ETH node - does that work?

Reply:
Yes, because we are following the same instructions that his tutorial contains: https://geth.ethereum.org/docs/getting-started/dev-mode it says `Geth has a development mode that sets up a single node Ethereum test network with options optimized for developing on local machines.`